recipes/autoconf-archive/all/conandata.yml

```diff
@@ -0,0 +1,10 @@
+sources:
+  "2018.03.13":
+    url: "https://ftpmirror.gnu.org/autoconf-archive/autoconf-archive-2018.03.13.tar.xz"
+    sha256: "6175f90d9fa64c4d939bdbb3e8511ae0ee2134863a2c7bf8d9733819efa6e159"
+  "2019.01.06":
+    url: "https://ftpmirror.gnu.org/autoconf-archive/autoconf-archive-2019.01.06.tar.xz"
+    sha256: "17195c833098da79de5778ee90948f4c5d90ed1a0cf8391b4ab348e2ec511e3f"
+  "2021.02.19":
```

Comment:
I think adding only the most recent collection is enough.
Also, make sure to add a final newline.

Reply:
Should I remove the old ones? I guess there's no harm in keeping them (except the CI has to build packages for them). Also I added the missing newlines but the bot doesn't think so

Reply:
ignore the last sentence please

Reply:
I suppose the latest release is a superset of the older releases + extra fixes.
A new recipe on cci will always use the latest.
If an older release is really needed, we can always add it afterwards.